rtcpeerconnection.go

```diff
@@ -762,6 +786,46 @@ func (pc *RTCPeerConnection) Close() error {
 	return nil
 }
 
+func (pclog *RTCPeerLogger) Debug(debug string){
+	pclog.DEBUG.Println(debug)
+}
+
+func (pclog *RTCPeerLogger) Error(error string){
+	pclog.ERROR.Println(error)
+}
+
+func (pclog *RTCPeerLogger) Info(info string){
+	pclog.INFO.Println(info)
+}
+
+func (pclog *RTCPeerLogger) Warning(warning string){
+	pclog.WARNING.Println(warning)
+}
+
+func (pclog *RTCPeerLogger) Trace(trace string){
+	pclog.TRACE.Println(trace)
+}
+
+func (pclog *RTCPeerLogger) setupLogger(logLevel int) {
```

Comment:
Can you make `logLevel` an enum?

Users might be confused by just passing an integer. Would be nice if they could pass `Trace` `Debug` `Info` or `Warning`

Reply:
Okay so I changed the function name to setLogLevel and it accepts a logging level enum.  I am increasing the number of loggers that can be used according to the logging level, but if you think that the logging level should just be the type of logger you can use I can change that.